linux/check_inbox_driver/get_xorg_version.yml

```diff
@@ -52,21 +60,18 @@
             - vmware_video_package_result.rc == 0
             - vmware_video_package_result.stdout is defined
             - vmware_video_package_result.stdout
-      when: guest_os_family in ["Debian", "Astra Linux (Orel)"]
 
-    - block:
-        - include_tasks: ../utils/get_installed_package_info.yml
+    - name: "Collect VMware video driver version"
+      when: video_driver_pkg_name
+      block:
+        - name: "Get information of video driver package '{{ video_driver_pkg_name }}'"
+          include_tasks: ../utils/get_installed_package_info.yml
           vars:
             package_name: "{{ video_driver_pkg_name }}"
 
         - name: "Get the VMware video driver version"
           ansible.builtin.set_fact:
-            inbox_drivers_versions: "{{ inbox_drivers_versions | combine({'xf86-video-vmware': package_info.Version.split(':')[-1] }) }}"
+            inbox_drivers_versions: "{{ inbox_drivers_versions | combine({'xf86-video-vmware': package_info.Version.split(':')[-1]}) }}"
```

Comment:
Only combine 'xf86-video-vmware'? how about other values of 'video_driver_pkg_name'?

Reply:
It is to collect video driver version from the package version. What else do we need here?

Reply:
I saw above has `{%- if guest_os_family == "RedHat" -%}xorg-x11-drv-vmware` setting for parameter `video_driver_pkg_name`, but no 'combine' value.

Reply:
`xorg-x11-drv-vmware` is the package name. Here `xf86-video-vmware` represents xorg video driver. I can replace `xf86-video-vmware` with `xorg-vmware-video-driver` here.